docs/helm-deployment-aws_cloud.md

```diff
@@ -385,16 +385,207 @@ helm status $ACSRELEASE
 
 * You can access all components of Alfresco Content Services using the same root address, but different paths:
 ```bash
-  Content: https://$EXTERNALHOST/alfresco
   Share: https://$EXTERNALHOST/share
-  Solr: https://$EXTERNALHOST/solr
+  Content: https://$EXTERNALHOST/alfresco
 ```
 
 To see the full list of values that were applied to the deployment, run:
 ```bash
 helm get values -a $ACSRELEASE
 ```
 
+### Network Hardening
+
+The `kops` utility will setup platform for setting up ACS cluster, but with some extra steps the networking on AWS infrastructure can be hardened.  
+
+See AWS documentation: https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_SecurityGroups.html
+
+Below are some recommended modifications to Security Groups that manage Inbound and Outbound traffic.
+
+
+#### Lockdown Bastion 
+
+By default, SSH access to bastion host is open from everywhere for Inbound and Outbound traffic.  You may want it to lock down to a specific IP address(es).
```

Comment:
Suggestion: By default, SSH access to _the_ bastion host is open from everywhere for inbound and outbound traffic.  You may want to lock _it_ down to a specific IP address(es).